crates/driver/src/tests/cases/order_prioritization.rs

```diff
@@ -37,6 +91,202 @@ async fn sorting() {
     test.solve().await.ok();
 }
 
+/// The most recent orders get prioritized no matter other factors.
+#[tokio::test]
+#[ignore]
+async fn creation_timestamp_sorting() {
+    let now = Utc::now().timestamp() as u32;
+    let solver = test_solver().fee_handler(FeeHandler::Driver);
+    let test = setup()
+        .solvers(vec![solver.clone()])
+        .pool(ab_pool())
+        .order_priority_strategy(OrderPriorityStrategy::CreationTimestamp {
+            max_order_age: Some(Duration::from_secs(10)),
+        })
+        .order(
+            ab_order()
+                .created(now)
+                .reduce_amount("1e-3".ether().into_wei()),
+        )
+        .order(
+            ab_order()
+                .rename("2")
+                .limit()
+                .created(now - 1)
+                .reduce_amount("1e-2".ether().into_wei())
+                .quote(OrderQuote::default().solver(solver.address())),
+        )
+        .order(
+            ab_order()
+                .rename("3")
+                .created(now - 20)
+                .quote(OrderQuote::default().solver(solver.address())),
+        )
+        .solution(ab_solution())
+        .done()
+        .await;
+
+    test.solve().await.ok();
+}
+
+/// Market orders get prioritized no matter other factors.
```

Comment:
Given that we no longer have market orders, I wonder if we should actually remove this strategy to lighten the code we need to maintain a bit. Not blocking though.

Reply:
Makes sense. Dropped the `OrderClass` strategy.